contrib/builder/rpm/generate.sh

```diff
@@ -93,6 +106,39 @@ for version in "${versions[@]}"; do
 
 	echo >> "$version/Dockerfile"
 
+	# centos, fedora, & oraclelinux:7 do not have a libseccomp.a for compiling static dockerinit
+	# ONLY install libseccomp.a from source, this can be removed once dockerinit is removed
+	case "$from" in
+		opensuse:*|oraclelinux:6) ;;
+		*)
+			awk '$1 == "ENV" && $2 == "SECCOMP_VERSION" { print; exit }' ../../../Dockerfile >> "$version/Dockerfile"
+			cat <<-EOF >> "$version/Dockerfile"
+			RUN buildDeps=' \
+				automake \
+				libtool \
+			' \
+			&& set -x \
+			&& ${installer} install -y \$buildDeps \
+			&& export SECCOMP_PATH=\$(mktemp -d) \
+			&& git clone https://github.com/seccomp/libseccomp.git "\$SECCOMP_PATH" \
+			&& ( \
+				cd "\$SECCOMP_PATH" \
+				&& git checkout "\$SECCOMP_VERSION" \
+				&& ./autogen.sh \
+				&& ./configure --prefix=/usr \
+				&& make \
+				&& install -c src/.libs/libseccomp.a /usr/lib/libseccomp.a \
+				&& chmod 644 /usr/lib/libseccomp.a \
+				&& ranlib /usr/lib/libseccomp.a \
+				&& ldconfig -n /usr/lib \
+			) \
+			&& rm -rf "\$SECCOMP_PATH"
```

Comment:
Did you make a variable for `buildDeps` so they could have an `apt-get purge --auto-remove $buildDeps` down here, too?